thehive-templates/Threatcrowd_1_0/long.html

```diff
@@ -0,0 +1,69 @@
+<div class="panel panel-info" ng-if="success">
+    <div class="panel-heading">
+        Threatcrowd analysis for 
+        <strong>{{artifact.data}}</strong>
+    </div>
+    <div class="panel-body">
```

Comment:
This should catch an empty result, because currently there would be an empty table.